yarn-project/Earthfile

```diff
@@ -36,7 +36,7 @@ build:
     BUILD ../barretenberg/cpp/+build
     BUILD ../barretenberg/ts/+build
     BUILD ../noir/+nargo
-    BUILD ../noir-projects/+build
+    BUILD ../noir-projects/+build --RAYON_NUM_THREADS=$RAYON_NUM_THREADS
```

Comment:
damn it really be like that huh

Reply:
It shouldn't be

Reply:
args by default act like opt-in env vars

Reply:
Might be cleaner to use `--pass-args`